src/studio/src/designer/backend/Views/Home/StartPage.cshtml

```diff
@@ -7,6 +7,10 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta property="og:title" content="Altinn Studio" />
+  <meta property="og:type" content="website" />
+  <meta property="og:image" content="~/designer/img/altinn-studio-social-preview.png" />
```

Comment:
Where do these pngs come from?
Could we get them as svg?

Reply:
This comes from @altinnadmin, and I'm guessing its not available as an svg 😅 
Other PNGs come from the figma page for the logos. I have replaced png in repositories solution with svg as suggested.

Reply:
@mjulstein Yup, created it as [social preview](https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/customizing-your-repositorys-social-media-preview) image for the altinn-studio repo on github.
Not sure why github does not support svg, might be that not all social platforms supports it.